src/ResourceManager/KeyVault/Commands.KeyVault/Commands.KeyVault.csproj

```diff
@@ -237,4 +278,11 @@
     </Content>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-</Project>
+  <Import Project="..\..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('..\..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
```

Comment:
Microsoft.Azure.Common is still required - 

PSVaultIdentityItem.cs(30,125,30,127): error CS0012: The type 'ResourceBaseExtended' is defined in an assembly that is not referenced. You must add a reference to assembly 'Microsoft.Azure.Common, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.